docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -208,7 +208,7 @@ The following properties are supported.
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
-|quarkus.http.limits.max-body-size| `unlimited` |The maximum size of request body.
+|quarkus.http.limits.max-body-size| `10240K` |The maximum size of request body.
 |quarkus.http.limits.max-header-size|`2OK`|The maximum length of all headers.
```

Comment:
Unrelated to this PR, but this looks like a typo. It should have been `20K` instead of `2OK`. i.e. the digit `0` instead of the capital letter `O`

Reply:
Ouch... :)

Reply:
Ah, yes :)

Reply:
Will this be a blocker to merging the pull request?

Reply:
Nope, you can fix it in another PR if you like

Reply:
Sure, I will raise another PR for it :)